assets/_scss/elements/_buttons.scss

```diff
@@ -29,78 +32,123 @@ input[type="image"] {
 
   &:hover,
   &.usa-button-hover {
-    background-color: #616161;
+    background-color: $color-primary-darker;
   }
 
   &:focus,
   &.usa-button-focus {
-    box-shadow: 0 0 0 2px $color-focus;
+    box-shadow: $focus-shadow;
   }
+
   &:active,
   &.usa-button-active {
-    background-color: #000;
+    background-color: $color-primary-darkest;
   }
 
   @media (min-width: $small-screen) {
     width: auto;
-  }  
-}
+  }
 
-input[type="submit"]:disabled,
-.usa-button-disabled {
-  background-color: #e0e0e0;
-  color: #616161;
-  cursor: default;
+  &.usa-button-primary-alt {
+    background-color: $color-primary-alt;
+    color: $color-base;
 
-  &:hover {
-    background-color: #e0e0e0;
+    &:hover,
+    &.usa-button-hover {
+      background-color: $color-primary-alt-dark;
+    }
+
+    &:active,
+    &.usa-button-active {
+      background-color: $color-primary-alt-darkest;
+      color: $color-white;
```

Comment:
Can you remove the focus and active state?

<img width="202" alt="screen shot 2015-08-14 at 6 20 55 pm" src="https://cloud.githubusercontent.com/assets/5249443/9308661/3b306b16-44ba-11e5-86b4-85166c8478ad.png">